docs/wiki/EL/data-structures.md

```diff
@@ -126,16 +126,51 @@ The structure `T` consists of the following:
 - **from** – The address of the sender, that will be signing the transaction. This must be an externally-owned account as contract accounts cannot send transactions.
 - **to**: Address of an account to receive funds, or zero for contract creation.
 - **value**: amount of ETH to transfer from sender to recipient.
-- **input data** – optional field to include arbitrary data
+- **input data**: optional field to include arbitrary data
 - **data**: Input data for a message call together with the message signature.
 - **(v, r, s)**: Values encoding signature of a sender. Serves as identifier of the sender
 
 ### TODO: Explain Receipt Trie
+ 
+## World State Trie
 
-### TODO: Explain World State Trie
+The **World State Trie** is the core data structure that represents Ethereum's current state. It maps the keccak-256 hashed 20 byte account addresses to their RLP encoded states utilizing a **Merkle Patricia Trie** where the key-value pairs are stored as byte arrays to byte arrays in the leaves of the trie.
+
+Accounts can be thought of as either smart contract accounts or non-smart contract accounts, which are also known as **EOA's** (Externally Owned Accounts). **EOA's** are used to initiate transactions or trigger the execution of smart contract logic using an associated private key whereas contract accounts execute the contracts when called.
```

Comment:
> **EOA's** are used to initiate transactions or trigger the execution
Its not an either or case. Transaction is ALWAYS initiated, which _may_ cause an code execution

Suggested:
Accounts can be categorized as either smart contract accounts with code or Externally Owned Accounts (EOAs) associated with private keys. EOAs are used to initiate transactions with other EOAs or smart contract accounts, triggering the execution of the associated contract code.

Reply:
great rec @raxhvl! updated in latest commit